js/scripts.js

```diff
@@ -0,0 +1,18 @@
+$(document).ready(function() {
+	function Button() {
```

Comment:
missing argument text in function Button, you want to assign `text` to `this.text` but `text` doesn't exist